e2e-xvfb.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+nohup /usr/bin/Xvfb :99 -ac -screen 0 1280x1024x24 &
+export DISPLAY=:99
+yarn
+npm rebuild node-sass
```

Comment:
Well, not sure about that. We might actually need `yarn webdriver-manager update`.